models/intel/human-pose-estimation-0001/description/human-pose-estimation-0001.md

```diff
@@ -35,7 +37,7 @@ Tested on a COCO validation subset from the original paper: Cao et al. ["Realtim
 
 ## Outputs
 
-1. The net outputs two blobs with shapes: [1, 38, 32, 57] and [1, 19, 32, 57]. The first blob contains keypoint pairwise relations (part affinity fields), the second one contains keypoint heatmaps.
+The net outputs two blobs with the [1, 38, 32, 57] and [1, 19, 32, 57] shapes. The first blob contains keypoint-pairwise relations (part affinity fields), while the second blob contains keypoint heatmaps.
```

Comment:
> keypoint-pairwise relations

I don't think that's correct. They are not relations that are "keypoint-pairwise", they are pairwise relations of keypoints.